lib/java/directory-watcher-wrapper/src/main/java/module-info.java

```diff
@@ -6,4 +6,5 @@
 
   exports io.methvin.watcher;
   exports io.methvin.watchservice;
+  exports io.methvin.watchservice.jna;
```

Comment:
Looking at the [error message](https://github.com/enso-org/enso/actions/runs/11051733295/job/30702270581#step:7:6365) the call is coming from:
```
com.sun.jna.Klass.newInstance(Klass.java:52)
```
which is probably using reflection. As such it should be enough to just:
```java
opens io.methvin.watchservice.jna;
```

Reply:
I thought that `opens` implies `exports`? But sure, let's try that - 17641f325ce6be2b4a77eb880dbb2fa725478a72